src/main/java/org/folio/circulation/domain/Item.java

```diff
@@ -23,8 +24,11 @@
 import io.vertx.core.json.JsonObject;
 import lombok.Getter;
 import lombok.NonNull;
+import lombok.ToString;
 
+@ToString(onlyExplicitlyIncluded = true)
 public class Item {
+  @ToString.Include
   private final String id;
   private final JsonObject itemRepresentation;
```

Comment:
I think it will be better to include itemRepresentation instead of id

Reply:
Done